Dockerfile

```
@@ -0,0 +1,15 @@
+FROM ubuntu:22.04
+
+COPY inittimezone /usr/local/bin/inittimezone
+RUN chmod +x /usr/local/bin/inittimezone
+CMD ./usr/local/bin/inittimezone
+
+RUN apt-get update -y && apt-get install -y build-essential cmake git wget vim libboost-all-dev libdeal.ii-dev
+
+RUN mkdir exercise && cd exercise
+RUN git clone https://github.com/jbeder/yaml-cpp ./yaml-cpp
+RUN mkdir ./yaml-cpp/build && cd ./yaml-cpp/build && cmake .. && make && make install
+RUN cd .. && git clone https://github.com/Congpy/cmake-exercise-wt2223.git
+RUN mkdir ./build && cd ./build && cmake ../ && make
```

```suggestion
RUN git clone https://github.com/Congpy/cmake-exercise-wt2223.git
RUN cd cmake-exercise-wt2223 && mkdir ./build && cd ./build && cmake ../ && make
```
Every `RUN` command starts from root again.

---

Yes I have formatted Windows 11 and installed it in HDD rather than SSD, thank you ! :)